app/check_tx.go

```diff
@@ -54,6 +54,8 @@ func (app *App) CheckTx(req *abci.RequestCheckTx) (*abci.ResponseCheckTx, error)
 		if err != nil {
 			return responseCheckTxWithEvents(err, 0, 0, []abci.Event{}, false), err
 		}
+		// Cache the validation result for ProcessProposal to reuse
+		app.txValidationCache.Set(btx.Tx)
```

Comment:
[question] how does this save the validation result? It looks like it just saves the tx and not the result of ValidateBlobTx.

Perhaps the tx validation cache only stores keys of txs that are valid? If yes, can that be clarified in a comment on the tx validation cache?

Reply:
Yes, we only store if the tx is valid, that is correct, I will indicate this in comments, thanks